common/taskana-common/src/main/resources/sql/postgres/taskana_schema_update_4.3.0_to_4.8.0_postgres.sql

```diff
@@ -5,3 +5,43 @@ SET search_path = %schemaName%;
 INSERT INTO TASKANA_SCHEMA_VERSION (VERSION, CREATED) VALUES ('4.8.0', CURRENT_TIMESTAMP);
```

Comment:
please change the comment in line 1, this script updates more than the comment suggests. Same for the other update scripts

Reply:
ok